browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -2570,7 +2573,10 @@ If you don't accept this request, VPN will not reconnect and your internet conne
       ERC20Approve
     </message>
     <message name="IDS_WALLET_TX_INFO_SENT" desc="Brave wallet sent assets tx history">
-      <ph name="ACCOUNT_NAME">%1$s<ex>Account1</ex></ph> sent <ph name="CRYPTO_VALUE">%2$s<ex>0.0444</ex></ph> <ph name="ASSET">%3$s<ex>ETH</ex></ph> ($<ph name="FIAT_VALUE">%4$s<ex>10.00</ex></ph>) - <ph name="DATE">%5$s<ex>2021-11-12 08:38pm</ex></ph>
+      <ph name="ACCOUNT_NAME">%1$s<ex>Account1</ex></ph> sent <ph name="CRYPTO_VALUE">%2$s<ex>0.0444</ex></ph> <ph name="ASSET">%3$s<ex>ETH</ex></ph> - <ph name="DATE">%4$s<ex>2021-11-12 08:38pm</ex></ph>
```

Comment:
why did we change that format? We don't want to show fiat value anymore?

Reply:
The fiat value is not shown on desktop, or does Android use a different format?

Reply:
I don't remember the history. But I think it was in the original Android spec. Let's don't change it and we will have another issue if we think it has to be a parity between 2 platforms

Reply:
I see, I have reverted this part.